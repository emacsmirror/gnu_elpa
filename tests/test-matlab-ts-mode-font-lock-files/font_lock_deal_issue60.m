% -*- matlab-ts -*-

% See https://github.com/acristoffers/tree-sitter-matlab/issues/60

person = struct('Age', {30, 25, 15}, 'City', {'c1', 'c2', 'c3'});
[person([1:3,1]).names] = deal('no-name');
