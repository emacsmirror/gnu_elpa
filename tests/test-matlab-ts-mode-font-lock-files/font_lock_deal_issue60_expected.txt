c ccc ccccccccc ccc

c ccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

vvvvvv o BBBBBBbSsssSD bnnD nnD nnbD SssssSD bSssSD SssSD SssSbbD
bddddddbbnDnDnbbDdddddb o BBBBbSsssssssSbD
